i6_models/assemblies/conformer/conformer_v1.py

```diff
@@ -0,0 +1,118 @@
+from __future__ import annotations
+
+__all__ = ["ConformerBlockV1Config", "ConformerEncoderV1Config", "ConformerBlockV1", "ConformerEncoderV1"]
+
+import torch
+from torch import nn
+from dataclasses import dataclass
+from typing import Tuple
+
+from i6_models.config import ModelConfiguration, ModuleFactoryV1
+from i6_models.parts.conformer import (
+    ConformerConvolutionV1,
+    ConformerConvolutionV1Config,
+    ConformerMHSAV1,
+    ConformerMHSAV1Config,
+    ConformerPositionwiseFeedForwardV1,
+    ConformerPositionwiseFeedForwardV1Config,
+)
+
+
+@dataclass
+class ConformerBlockV1Config(ModelConfiguration):
+    """
+    Attributes:
+        ff_cfg: Configuration for ConformerPositionwiseFeedForwardV1
+        mhsa_cfg: Configuration for ConformerMHSAV1
+        conv_cfg: Configuration for ConformerConvolutionV1
+    """
+
+    # nested configurations
+    ff_cfg: ConformerPositionwiseFeedForwardV1Config
+    mhsa_cfg: ConformerMHSAV1Config
+    conv_cfg: ConformerConvolutionV1Config
```

Comment:
```suggestion
    ff: ConformerPositionwiseFeedForwardV1Config
    mhsa: ConformerMHSAV1Config
    conv: ConformerConvolutionV1Config
```
I thought this was already discussed somewhere but I currently don't find it anymore. Isn't it nicer like this?

Reply:
I would say no. For me `ff` would imply that this is the FF module or some factory (like for Norm in the Convolution).
If more people prefer the shorter variation we can change it, otherwise I would leave it for now.

Reply:
My argument is not just to have it shorter, but also for consistency. The `_cfg` postfix describes the type of the attrib. For other attribs, we don't have that (e.g. `num_layers_int`).

Reply:
I would argue here that for `num_layers` the analogy is that you put `num` and not just `layers`, since in our case we do not pass the `ff` layer but a config for the `ff` layer. But I would go with the majority here, I think both variants are okay.

Reply:
I would also prefer `cfg`. it does not add too much and makes a clear distinction between layer/module and config

Reply:
I also prefer `_cfg`